server/app/views/admin/apikeys/ApiKeyNewOneView.java

```diff
@@ -32,8 +35,33 @@ public final class ApiKeyNewOneView extends BaseHtmlView {
   private final Messages enUsMessages;
   private final Slugify slugifier = new Slugify();
 
+  private static final String EXPIRATION_DESCRIPTION =
+      "Specify a date when this API key will no longer be valid. The expiration date"
+          + " may be edited after the API key is created. It is strongly recommended to"
+          + " allow API keys to expire and create new ones to ensure that if a key"
+          + " has been stolen, malicious users will eventually lose access. For"
+          + " highly sensitive data, expiring keys once a week or once a month is"
+          + " recommended. Once a year is acceptable in most situations.";
+  private static final DomContent[] SUBNET_DESCRIPTION = {
+    text("Specify a subnet using "),
+    new LinkElement()
+        .setText("CIDR notation")
+        .setHref("https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing")
+        .opensInNewTab()
+        .asAnchorText(),
+    text(
+        " that is allowed to call the API. The"
```

Comment:
I think some of the text lines were deleted accidentally

Reply:
looks right to me?

Reply:
yeah, I'd deleted. I didn't see the construction just above.  But GH is duplicating my comments today.... :/